### PR TITLE
mircat: Fix node ID type

### PR DESCRIPTION
**Pull request by sergefdrv**

### CODE REVIEW

File: cmd/mircat/eventhandler.go [hunk 1]
```diff
@@ -4,19 +4,20 @@ package main
...
+	"github.com/filecoin-project/mir/pkg/eventlog"
+	"github.com/filecoin-project/mir/pkg/pb/eventpb"
+	"github.com/filecoin-project/mir/pkg/pb/isspb"
+	"github.com/filecoin-project/mir/pkg/pb/recordingpb"
 )
 
 type eventMetadata struct {
 	time   int64
-	nodeID uint64
+	nodeID string
```
Comment (matejpavlovic) [inline cmd/mircat/eventhandler.go:20]: I'd go for abstract `t.NodeID` here

File: cmd/mircat/prettydisplay.go [hunk 2]
```diff
@@ -15,7 +16,7 @@ func getMetaTag(eventType string, metadata eventMetadata) string {
 	return fmt.Sprintf("%s %s",
 		boldGreen.Style(fmt.Sprintf("[ Event_%s ]", eventType)),
 		boldCyan.Style(fmt.Sprintf("[ Node #%s ] [ Time _%s ] [ Index #%s ]",
-			strconv.FormatUint(metadata.nodeID, 10),
+			metadata.nodeID,
```
Comment (matejpavlovic) [inline cmd/mircat/prettydisplay.go:19]: `"[ Node %v ] ...", ..., metadata.nodeID, ...`
could be more general

**Comment by sergefdrv** [comment]
Summary of [changes](https://github.com/filecoin-project/mir/compare/182452e7f074fc4f946f56234cd2e4e9373b7b36..341b88d0c8edef291cb70d6e7db39b2364e8d77d): use abstract NodeID type

**Review by matejpavlovic — APPROVED** [review]
Looks good, thanks!

**Pull request merged**